yarn-project/pxe/src/pxe_oracle_interface/pxe_oracle_interface.test.ts

```diff
@@ -420,9 +425,12 @@ describe('PXEOracleInterface', () => {
     });
 
     it('should not sync tagged logs with a blockNumber > maxBlockNumber', async () => {
+      const maxBlockNumber = 1;
```

Comment:
This 1 is also super fragile, it just happens to match the value used in the setup. Make these constants.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/13152/commits/fe43da92095af7fb692a7f2b7f8f905258f01e5c